content/content/strings.md

```diff
@@ -45,7 +45,7 @@ Strings can also almost be thought of as `seq[char]` with respect to assignment
 
 [seqs]: /seqs/#immutability
 
-## A note about unicode
-Unicode symbols are allowed in strings, but are not treated in any special way, so if you want count glyphs or uppercase unicode symbols, you must use the `unicode` module.
```

Comment:
Unicode is referring to the `unicode` module inside the ` so it should be lowercase there.

Reply:
Good catch. Fixed.